CONTRIBUTING.md

```diff
@@ -167,9 +166,16 @@ cd packages/opentelemetry-module-name
 npm test
 ```
 
+To run the unit tests continuously in watch mode while developing, use:
+
+```sh
+# Run test in watch mode
+npm run tdd
+```
+
 ### Linting
 
-This project uses a combination of `gts` and `eslint`. Just like tests and compilation, linting can be done for all packages or only a single package.
+This project uses `eslint` to lint source codes. Just like tests and compilation, linting can be done for all packages or only a single package.
```

Comment:
```suggestion
This project uses `eslint` to lint source code. Just like tests and compilation, linting can be done for all packages or only a single package.
```

Feel free to ignore, but I think code in this context is a mass noun and does not have a plural. 
English is not my first language, though so there's a chance I'm wrong. :slightly_smiling_face: